libs/nest/aws/src/lib/aws.service.ts

```diff
@@ -31,44 +90,54 @@ export class AwsService {
       ContentEncoding?: string
     },
   ): Promise<string> {
-    const uploadParams = {
+    const command = new PutObjectCommand({
       Bucket: bucket,
       Key: fileName,
       Body: content,
       ...uploadParameters,
-    }
+    })
 
-    const { Location: url } = await this.s3.upload(uploadParams).promise()
-    return url
+    await this.s3Client.send(command)
+    return `https://${bucket}.s3.amazonaws.com/${fileName}`
```

Comment:
~Should we hardcode this here? Do S3 URLs depend on region and more?

Reply:
I asked around and it seems the bucket url is correct since s3 bucket names are unique and do not depend on the region.

Reply:
Also, IMO, we shouldn't hard-code _config values_, which the bucket name indeed is.